dotCMS/src/main/java/com/dotcms/rest/api/v1/system/websocket/SystemEventsWebSocketEndPoint.java

```diff
@@ -83,9 +82,25 @@ public void open(final Session session) {
 			}
 		}
 
-		if (addToNormalSession) {
+		if (!isLoggedIn) {
 
-			this.queue.add(session);
+			try {
+
+				final ForbiddenException forbiddenException = new ForbiddenException("A web socket connection requires a previous web session created");
+				if (session.isOpen()) {
+
+					session.getAsyncRemote().sendObject(forbiddenException);
+					session.close(new CloseReason(new ForbiddenCloseCode(),
```

Comment:
Why do we have a newly  ForbiddenCloseCode Object every time?, maybe is better is it is a class constant some like:

...
ForbiddenCloseCode  F = new ForbiddenCloseCode();
...
public void open(final Session session) {
    ...
    session.close(new CloseReason(F, ...
    ...
}